tests/test_compare.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import os
+from pychimera import patch_environ, enable_chimera
+
+patch_environ()
+# enable_chimera()
```

Comment:
Why is this commented out? It should be there. Are you having problems with initializing `pytest` within `pychimera`? If so, look at `gaudi` tests for some _inspiration_ ;)

Reply:
I think it was because when I was having troubles with importing chimera I was trying to understand the behavior of those statements and I commented out the `enable_chimera()` one at some point to do some checks. Because no error I received, I must have forgotten to uncomment out it. It is clear that it was not intentional.

Reply:
Ok, I see that now is working properly. Challenge:

- What do you think it would happen if you have more than one `test_*.py` file? `pychimera.patch_environ` reloads the Python interpreter as part of its behaviour, so that is totally going to affect your tests.

Recommendation: use `patch_environ` + `enable_chimera` _outside_ the test files. Maybe the `conftest.py` is a good starting point... or do as we are doing [in GaudiMM (creating a wrapper around the PyTest `main` call](https://github.com/insilichem/gaudi/blob/master/tests/run_test.py).

Reply:
I didn't have noticed any change in the execution of the test or tests (I tried it, also, creating new tests), I only realized that it was only necessary  to reloads the Python interpreter once. So I have added a new file, a run_test.py for the `pychimera.patch_environ`, to do it initially, before the tests and then run all the test files (although in this case it is only one test file) executing this file with `python run_test.py`.